docs/getting-started/run-superbench.md

```diff
@@ -0,0 +1,33 @@
+---
+id: run-superbench
+---
+
+# Run SuperBench
+
+Having prepared benchmark configuration and inventory files,
+it will be easy to run SuperBench over all managed nodes.
+
+## Deploy
+
+We need to deploy SuperBench environments to all managed nodes first,
+it will access all nodes, pull container image and prepare container.
```

Comment:
Leveraging SuperBench `deploy` command, we can easily deploy SuperBench environment to all managed nodes. After running the command, SuperBench will automatically access all nodes, pull container image and prepare container.